Dangerfile

```diff
@@ -28,14 +46,31 @@ db_files  = all_touched_files.select { |filepath| filepath.include? "db/" }
 app_files = all_touched_files.select { |filepath| filepath.include? "app/" }
 
 if !db_files.empty? && !app_files.empty?
-  msg = "Modified files in `db/` and `app/` inside the same PR!\n\n**db file(s)**"
-  db_files.each { |file| msg += "\n- #{file}" }
-  msg += "\n\n**app file(s)**"
-  app_files.each { |file| msg += "\n- #{file}" }
-  msg += "\n\nIt is recommended to make db changes in their own PR since migrations do not run automatically with vets-api deployments. Application code must always be backwards compatible with the DB, both before and after migrations have been run."
+  msg = <<~HTML
+    Modified files in `db/` and `app/` inside the same PR!
+
+    <details><summary>File Summary</summary>
+
+    #### db file(s)
+
+    - #{db_files.collect { |filepath| "#{filepath}" }.join("\n- ")} 
+
+    #### app file(s)
+
+    - #{app_files.collect { |filepath| "#{filepath}" }.join("\n- ")} 
+
+    </details>
+
+    Database migrations do not run automatically with vets-api deployments. Application code must always be backwards compatible with the DB, both before and after migrations have been run. For more info: 
+
+    - [guidance on safe db migrations](https://github.com/ankane/strong_migrations#checks)
+    - [`vets-api` deployment process](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/deployment.md)
+
+  HTML
```

Comment:
The diff looks big, but all I did was update this section to display the message using heredoc consistent with how we do it for max PR size. It helps with readability and easily allows us to include the collapsable `<details><summary>File Summary</summary>` html.

Also changed the wording around.